packages/gatsby-theme-newrelic/src/components/Tabs/Tabs.js

```diff
@@ -30,6 +32,38 @@ const Tabs = ({ children, initialTab }) => {
     }
   };
 
+  const location = useLocation();
+
+  // this needs to run in a useEffect since the hash
+  // isn't available on the server during SSG.
+  // to put it another way, in order to have this work with SSG
+  // we'd hypothetically need to render a separate HTML file
+  // for every single tab id on the page.
+  // i don't think we can do this with Gatsby.
+  useEffect(() => {
+    const hash = location.hash.replace('#', '');
+    if (hash !== '') {
+      const [_tabBar, tabPages] = children;
+      // Tabs should always look like this:
+      // ```
+      // <Tabs>
+      //   <Tabs.Bar>...</Tabs.Bar>
+      //   <Tabs.Pages>...</Tabs.Pages>
+      // </Tabs>
+      // ```
+      // if not, something is wrong and we'll only break the page
+      // by trying further.
+      if (tabPages.type === Pages) {
+        const pages = tabPages.props.children;
+        const index = pages.findIndex((page) => page.props.id === hash);
+        if (index !== -1) {
+          setTab(index);
+          tabsContainer.current.scrollIntoView();
```

Comment:
Having a bit of an issue with the `scrollIntoView()` as we're not able to get around the header height here, might need the ole:
```
const y = element.getBoundingClientRect().top + window.pageYOffset + yOffset;

window.scrollTo({top: y, behavior: 'smooth'});
```